tests/cpp/testobject/PdxType.hpp

```diff
@@ -569,22 +569,22 @@ class TESTOBJECT_EXPORT PdxType : public PdxSerializable {
     m_objectArrayEmptyPdxFieldName->push_back(
         std::shared_ptr<Address>(new Address(10, "street9", "city9")));
 
-    m_byte252 = new int8_t[252];
+    m_byte252 = std::vector<int8_t>(252);
```

Comment:
The below can be replaced with:
```
 m_byte252 = std::vector<int8_t>(252, 0);
```
The second argument gets copied 252 times into the vector.

Reply:
Ah, I think what I was trying to do was get rid of the for loop below. But I forgot to delete it.  Ooops...

Reply:
Oops, seems like my brain is not 100%.. Yes,  I'll make that change.  Your suggestion is much better than a for-loop.